e2e/playwright.config.ts

```diff
@@ -17,33 +16,33 @@ export default defineConfig({
   /* Opt out of parallel tests on CI. */
   reporter: [
     ['list'],
-    [
-      '@alex_neo/playwright-azure-reporter',
-      {
-        orgUrl: process.env.AZURE_DEV_URL,
-        token: process.env.AZURE_DEVOPS_TOKEN,
-        planId: 27408,
-        projectName: '121 Platform',
-        environment: 'AQA',
-        logging: true,
-        testRunTitle: 'Playwright Test Suite',
-        publishTestResultsMode: 'testRun',
-        uploadAttachments: true,
-        attachmentsType: ['screenshot', 'video', 'trace'],
-        testRunConfig: {
-          owner: {
-            displayName: 'Krajewski, Piotr',
-          },
-          comment: 'Playwright Test Suite',
-          configurationIds: [],
-        },
-      } as AzureReporterOptions,
-    ],
+    // [
+    //   '@alex_neo/playwright-azure-reporter',
+    //   {
+    //     orgUrl: process.env.AZURE_DEV_URL,
+    //     token: process.env.AZURE_DEVOPS_TOKEN,
+    //     planId: 27408,
+    //     projectName: '121 Platform',
+    //     environment: 'AQA',
+    //     logging: true,
+    //     testRunTitle: 'Playwright Test Suite',
+    //     publishTestResultsMode: 'testRun',
+    //     uploadAttachments: true,
+    //     attachmentsType: ['screenshot', 'video', 'trace'],
+    //     testRunConfig: {
+    //       owner: {
+    //         displayName: 'Krajewski, Piotr',
+    //       },
+    //       comment: 'Playwright Test Suite',
+    //       configurationIds: [],
+    //     },
+    //   } as AzureReporterOptions,
+    // ],
```

Comment:
Why these are commented out? Means we can delete them or we are going to enable this section later?